ask.js

```diff
@@ -0,0 +1,16 @@
+const readline = require('readline');
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+
+function ask(questionText) {
+    return new Promise((resolve, reject) => {
+        rl.question(questionText, resolve);
+    });
+}
+
+
+
+
+
+//share file within project
+module.exports = { ask }
```

Comment:
This is a great start on modularization.  Keeping functions in their own file makes the code more digestible, and makes debugging a lot easier.  I saw in your notes that you were having trouble moving some game functions out of the game, and that could be for a multitude of reasons.  Having a global gamestate that gets updated by values coming back from the function might be one way to solve it though.